autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -67,19 +67,27 @@ impl UnidentifiedClient {
         );
 
         // Ignore invalid uaids (treat as None) so they'll be issued a new one
-        let uaid = uaid.as_deref().and_then(|uaid| Uuid::try_parse(uaid).ok());
+        let original_uaid = uaid.as_deref().and_then(|uaid| Uuid::try_parse(uaid).ok());
 
         let GetOrCreateUser {
             user,
             existing_user,
             flags,
-        } = self.get_or_create_user(uaid).await?;
+        } = self.get_or_create_user(original_uaid).await?;
         let uaid = user.uaid;
         debug!(
             "💬UnidentifiedClient::on_client_msg Hello! uaid: {} existing_user: {}",
             uaid, existing_user,
         );
-        let _ = self.app_state.metrics.incr("ua.command.hello");
+        self.app_state
+            .metrics
+            .incr_with_tags("ua.command.hello")
+            .with_tag("existing_user", &original_uaid.is_some().to_string())
+            .with_tag(
+                "reassigned",
+                &(original_uaid.unwrap_or(uaid) != uaid).to_string(),
+            )
+            .send();
```

Comment:
With all reassigned being new users I think one tag should cover it all instead, e.g. `uaid`: `existing` | `new` | `reassigned`

Reply:
I think you meant to say that all reassigned users were existing users, but yeah, I'll collapse these into one tag.

Reply:
Nope: they would be new (`existing_user` would be false), their new uaids generated from `Default::default()` in `get_or_create_user`'s fallback case